cli/azd/pkg/azure/arm_template.go

```diff
@@ -145,6 +145,7 @@ type AzdMetadata struct {
 	AutoGenerateConfig *AutoGenInput    `json:"config,omitempty"`
 	DefaultValueExpr   *string          `json:"defaultValueExpr,omitempty"`
 	Default            *string          `json:"default,omitempty"`
+	UsageName          *string          `json:"usageName,omitempty"`
```

Comment:
Can we change it to an array. So folks can use it for one or more. Since some templates do more than one model being provisioned in the same location?